examples/kubernetes/samba-ad-server-deployment.yml

```diff
@@ -0,0 +1,24 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: samba-ad-server
+  labels:
+    app: samba
```

Comment:
Should this be 'samba-ad' for consistency?

Reply:
updated